src/components/InfoModal.vue

```diff
@@ -176,4 +183,10 @@ export default {
 	background-size: 45%;
 	cursor: pointer;
 }
+
+.ps {
+	font-size: 0.8em;
+	text-wrap: balance;
+	text-align: center;
```

Comment:
In general, centered text is harder to read the aligned one.

```suggestion
	text-align: start;
```